scripts/release.js

```diff
@@ -1,77 +0,0 @@
-const cp = require('child_process');
```

Comment:
It seems our release file is called `bkRelease`, do we want to change it to `release`?

Reply:
Renamed 👌

Reply:
We'll need to edit the steps once this is merged.